pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -270,8 +253,7 @@ private TransferableBlock consumeAggregation() {
         new FunctionContext(FunctionContext.Type.AGGREGATION, functionName, arguments), true);
   }
 
-  private static AggregationFunction<?, ?> getAggFunctionForIntermediateInput(RexExpression.FunctionCall functionCall,
-      Map<Integer, Literal> literalArguments) {
+  private static AggregationFunction<?, ?> getAggFunctionForIntermediateInput(RexExpression.FunctionCall functionCall) {
```

Comment:
After the change do we still need to differentiate raw input and intermediate input?